specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/AccountFilters.json

```diff
@@ -199,6 +198,18 @@
         }
       ],
       "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true
+        },
```

Comment:
these (name, id , type) must be part of the common definition for ProxyResource

Reply:
same feedback for other files

---
In reply to: [387329390](https://github.com/Azure/azure-rest-api-specs/pull/8297#discussion_r387329390) [](ancestors = 387329390)